gm_dotnet_managed/GmodNET/GameConsoleWriter.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+
+public class GameConsoleWriter : TextWriter
+{
+    private class Tier0
+    {
+        [DllImport("tier0", CallingConvention = CallingConvention.Cdecl)]
+        public static extern void Msg([MarshalAs(UnmanagedType.LPStr)] string msg);
+    }
+
+    private static GameConsoleWriter instance;
+    private uint references;
+
+    public override string NewLine { get => "\n"; }
```

Comment:
You said to me in private conversation that it works for Windows, but is it a case for Linux and macOS?

Reply:
Never tried

Reply:
> A string containing "\r\n" for non-Unix platforms, or a string containing "\n" for Unix platforms.
~msdn
Using `\r\n` causes strange output in game console.
I think Source Engine uses `\n` without care about platform.
`Environment.NewLine` mostly needed for `Console` but not **Game**Console